tests/components/plugwise/test_climate.py

```diff
@@ -41,6 +44,44 @@ async def test_adam_climate_entity_attributes(hass, mock_smile_adam):
     assert attrs["preset_mode"] == "asleep"
 
 
+async def test_adam_climate_adjust_negative_testing(hass, mock_smile_adam):
+    """Test exceptions of climate entities."""
+    mock_smile_adam.set_preset.side_effect = PlugwiseException
+    mock_smile_adam.set_schedule_state.side_effect = PlugwiseException
+    mock_smile_adam.set_temperature.side_effect = PlugwiseException
+    entry = await async_init_integration(hass, mock_smile_adam)
+    assert entry.state == ENTRY_STATE_LOADED
+
+    await hass.services.async_call(
+        "climate",
+        "set_temperature",
+        {"entity_id": "climate.zone_lisa_wk", "temperature": 25},
+        blocking=True,
+    )
+    state = hass.states.get("climate.zone_lisa_wk")
+    attrs = state.attributes
+    assert attrs["temperature"] == 21.5
+
+    await hass.services.async_call(
+        "climate",
+        "set_preset_mode",
+        {"entity_id": "climate.zone_thermostat_jessie", "preset_mode": "home"},
+        blocking=True,
+    )
+    state = hass.states.get("climate.zone_thermostat_jessie")
+    attrs = state.attributes
+    assert attrs["preset_mode"] == "asleep"
+
+    await hass.services.async_call(
+        "climate",
+        "set_hvac_mode",
+        {"entity_id": "climate.zone_thermostat_jessie", "hvac_mode": HVAC_MODE_AUTO},
+        blocking=True,
+    )
+    state = hass.states.get("climate.zone_thermostat_jessie")
+    attrs = state.attributes
```

Comment:
was it intended to assert something here?

Reply:
ill go ahead and approve this, just double check this test in your usb branch or next PR.

Reply:
@ctalkington will do in the usb-branch coming up